dispatcher/stateful.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function StatefulDispatcher(dispatcher, data, onData) {
```

Comment:
It would be nice to have a one-liner that describes for what a static dispatcher could be used.